development/workflow.rst

```diff
@@ -0,0 +1,667 @@
+For more general information, you can consult the `LSST JIRA wiki <https://confluence.lsstcorp.org/display/JIRAatLSSTUserGuide/JIRA+at+LSST+User%27s+Guide+Home>`_ and Atlassian's docs for `JIRA <https://confluence.atlassian.com/jirasoftwarecloud/jira-software-documentation-764477791.html>`_ and `JIRA Agile <https://confluence.atlassian.com/agile067>`_.
+
+JIRA allows a myriad of metadata to be specified when creating a ticket; these are the most relevant fields:
+
+Project
+   This should be set to **Data Management**, unless you are creating a ticket for a different LSST subsystem.
+Issue Type
+   If the work is associated with an epic, the issue type is a 'Story.'
+   For emergent work, 'Bug' or 'Enhancement' can be used (see above for semantics).
+Summary
+   This is the ticket's title and should be written to help colleagues browsing JIRA dashboards.
+Components
+   You should choose from the pre-populated list of components to specify what part of the DM system the ticket relates to.
+   If in doubt, ask your T/CAM.
+Assignee
+   Typically you will assign yourself (or your T/CAM will assign you) to a ticket.
+   You can also assign tickets to others.
+   If you are uncertain about who the assignee should be you can allow the ticket to be automatically assigned (which defaults to the component's T/CAM; `RFC-51 <https://jira.lsstcorp.org/browse/RFC-51>`_).
+Description
+   For stories, you should specify the ticket's scope and outline your implementation design in this field.
+   For bug reports, include any information needed to diagnose and reproduce the issue.
+   Feel free to use `Atlassian markup syntax <https://jira.lsstcorp.org/secure/WikiRendererHelpAction.jspa?section=texteffects>`_. 
+Story Points
+   Use this field, at ticket creation time, to **estimate** the amount of effort involved to accomplish the work.
+   Keep in mind how *velocity* (see above) converts story points into real-world days.
+Labels
+   Think of labels as tags that you can use to sort your personal work.
+   Unlike the Component and Epic fields, you are free to create and use labels in any way you see fit.
+Linked Issues
+   You can express relationships between JIRA issues with this field.
+   For example, work that implements an RFC should link to that RFC.
+   You can also express dependencies to other work using a 'is Blocked by' relationship.
+Epic Link
+   If the ticket is a story, you must specify what epic it belongs to with this field.
+   By definition, bug or enhancement-type tickets are not associated with an epic.
+Team
+   You must specify which DM team doing the work with this field, for accounting purposes.
+   The owner of the epic should be consistent with the Team working on a ticket.
+
+.. _workflow-jira-ticket-status:
+
+Ticket status
+-------------
+
+Tickets are created with a status of 'Todo.'
+
+If a ticket was assigned to you by someone else, you can set the status to 'Acknowledged' once you agree to schedule the work.
```

Comment:
Indeed. I got this 'Acknowledged' workflow from the [JIRA at LSST User's Guide](https://confluence.lsstcorp.org/display/JIRAatLSSTUserGuide/Using+JIRA+on+LSST#UsingJIRAonLSST-IssueDetailScreenCustomization). I'll take it out.